### ISSUE TITLE
Add Radicle menu item on VCS menu

### ISSUE THREAD

**Issue by gsaslis** [cytechmobile/radicle-jetbrains-plugin#103]
The Version Control menu only shows up on projects where VCS hasn't been set up. 

It currently offers the option to share the project on github, but not on radicle, as shown in the screenshot below. 

![Screenshot 2022-10-25 at 8 42 46 AM](https://user-images.githubusercontent.com/2420882/197692194-af2a9ea2-9d97-427f-ba63-f8c76b44ffe5.png)

We should add a "Share on Radicle" option.

**Comment by Stelios123** [comment]
Share on radicle will not work here because the project is not git initialize

**Comment by gsaslis** [comment]
Exactly. When we come round to implementing this feature, our "Share on Radicle" should guide the user through: 
* git init
* rad init
* rad push

;)

**Comment by gsaslis** [comment]
@Stelios123 should we close this or was something not covered?

**Comment by gsaslis** [comment]
@Stelios123 bump ;)

**Comment by gsaslis** [comment]
closing - please reopen if there is something missing
